packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.css

```diff
@@ -17,3 +17,56 @@ li {
 .facet__value-checkbox {
   min-width: unset;
 }
+
+.facet__value_new-checkbox{
+  width: 20px;
+  height: 20px;
+  vertical-align: top;
+  border: solid 1px #cccccc;
+  border-radius: 4px;
+  margin-right: 6px;
```

Comment:
There is some [SLDS CSS classes](https://www.lightningdesignsystem.com/utilities/margin/) that we use when attributing margins, paddings... 
The rule is that before using any CSS value we first check if there is already an SLDS CSS class or an SLDS CSS variable that should give us the result we want.

Reply:
These css are added to make a div look like and act like a "input checkbox", removing the native "input checkbox". And thanks for the resource, will try to find suitable css styles from  [SLDS CSS classes](https://www.lightningdesignsystem.com/utilities/margin/)